src/main/asciidoc/license-epl2.adoc

```diff
@@ -0,0 +1,74 @@
+[subs="normal"]
```

Comment:
This is the Eclipse Foundation Specification License so the file name should be license-efsl.adoc.

Reply:
Yep, my mistake.  Will correct.